File: Client/src/Messages/FileNameMessage.cpp
#include "Message.h"



   void FileNameMessage::setFileName(string filename){
	   this->fileName=filename;
  }
  string FileNameMessage::getFileName(){
	  return fileName;
 }

  void FileNameMessage::setLastByte(char lastByte){
	  this->lastByte = lastByte;
 }
 char FileNameMessage::getLastByte(){
	  return lastByte;
}

File: Server/src/main/java/ameer/net/impl/rci/FileNameMessage.java
package ameer.net.impl.rci;
public class FileNameMessage extends Message {

	protected String fileName;
	protected byte lastByte='0';
	

  public void setFileName(String filename){
	   this.fileName=filename;
  }
 public String getFileName(){
	  return fileName;
 }
 
 public void setLastByte(byte lastByte){
	  this.lastByte = lastByte;
 }
public byte getLastByte(){
	  return lastByte;
}


}
